test/test_xxx_funcarg_last_comma.ok

```diff
@@ -1,2 +1,2 @@
-(function (F a))
-(function (G a))
+(function (F) (a))
+(function (G) (a))
```

Comment:
I wonder that this is breaking AST for function node?

Reply:
Changed.
from
`(function (F) (a))`
to
`(function (F (a)))`

Reply:
Update

`function! F(a, b = 2)` : `(function (F (a) (b 2)))`
`function! F(a = 1 + 1, b = 2)` : `(function (F (a (+ 1 1)) (b 2)))`
`function! F(a = 1, b = 2, ...)` : `(function (F (a 1) (b 2) . ...))`

Reply:
Ah, realized that to omit parenthesis if there's no default args resolves around backward compatibility. And that seems preferable.

Reply:
Update

`function! F(a, b = 2)` : `(function (F a (b 2)))`